app/views/shared/_two_columns.html.erb

```diff
@@ -1,36 +1,37 @@
-<% content_for :body do %>
-  <%= turbo_stream_from @conversation %>
+<%= turbo_stream_from @conversation %>
 
-  <div class="flex h-screen"
-      data-controller="click-toggle"
-      data-click-toggle-flippable-class="!hidden"
-  >
+<div class="flex h-screen"
+    data-controller="click-toggle"
+    data-click-toggle-flippable-class="!hidden"
+>
 
-    <div id="left-column"
-        class="flex-col h-screen w-[260px] bg-black hidden md:block overflow-y-auto overflow-x-clip scrollbar-hide"
-        data-click-toggle-target="destination"
-    >
+  <div id="left-column"
+      class="flex-col h-screen w-[260px] bg-black hidden md:block overflow-y-auto overflow-x-clip scrollbar-hide"
+      data-click-toggle-target="destination"
+  >
 
-      <div id="left-content-container" class="flex flex-col flex-grow">
-        <div class="pl-3 flex flex-col mb-28">
-          <div id="spacer-above-sticky-assistant" class="sticky top-0 h-4 z-10 bg-black"></div>
-          <%= render partial: "shared/assistant_sidebar_link", collection: @sidebar_assistants, as: :assistant %>
+    <div id="left-content-container" class="flex flex-col flex-grow">
+      <div class="pl-3 flex flex-col mb-28">
+        <div id="spacer-above-sticky-assistant" class="sticky top-0 h-4 z-10 bg-black"></div>
+        <%= render partial: "shared/assistant_sidebar_link", collection: @sidebar_assistants, as: :assistant %>
 
+        <turbo-frame id="conversations" target="conversation" data-controller="radio-behavior" data-radio-behavior-selected-class="bg-zinc-800 relationship-flex">
           <% @sidebar_conversations.each do |named_time_span, conversations| %>
             <div class="text-white text-opacity-40 text-xs mx-2 mt-10 mb-2">
               <%= named_time_span %>
             </div>
 
             <%= render partial: 'shared/conversation_sidebar_link', collection: conversations, as: :conversation %>
           <% end %>
-        </div>
-
-        <%= render partial: "shared/user_menu" %>
+        </turbo-frame>
       </div>
-    </div>
 
-    <div id="right-column" class="flex flex-1 flex-col h-screen bg-white dark:bg-graphite relative">
+      <%= render partial: "shared/user_menu" %>
+    </div>
+  </div>
 
+  <div id="right-column" class="flex flex-1 h-screen bg-white dark:bg-graphite">
+    <turbo-frame id="conversation" class="flex flex-1 flex-col relative">
```

Comment:
Hmm... yea, conceptually that does make more sense for this to live in index.html. However, I intentionally am wrapping the turbo-frame around messages + composer. I'm considering that whole right column to be the "conversation".

And the new.html includes the composer too. In place of the messages we'll eventually add the placeholder "tips for new conversation".

Reply:
If this were an application wide layout, it'd be clear to have the layout contain a left and right side, and the name of  the content boxes would be vague: "sidebar" and "content-area" or something.  If one page has the composer and message list in it, that's just one use of it. 

The way you have it here, the column layout is tied to the right hand side being for "messages," which means that any use of that which isn't messages is going to require a refactor or be a wedge. I'm fuzzy on what the "rest of" the site is going to look like, or if anything even needs to exist outside of this two column layout. 

Since it's all within messages, I don't think it matters that much.